src/app/renderer/container.ts

```diff
@@ -55,6 +57,8 @@ container.bind<SettingsStorageInterface>(SettingsStorageId).to(SettingsStorageIm
 container.bind<CollapseAllDirs>(CollapseAllDirs).toSelf().inSingletonScope();
 container.bind<ExpandAllDirs>(ExpandAllDirs).toSelf().inSingletonScope();
 container.bind<ToggleCollapseDir>(ToggleCollapseDir).toSelf().inSingletonScope();
+container.bind<SelectFilter>(SelectFilter).toSelf().inRequestScope();
+container.bind<ResetFilter>(ResetFilter).toSelf().inRequestScope();
```

Comment:
1. it would be nice if you sort the lines in the same way they are sorted on the file system (alphabetically)
2. should be `inSingletonScope`, not `inRequestScope`

Reply:
done